### PR TITLE
added purchase link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user feedback #7251 , so i added project plans and visio plans links. added highlights of Project plans and visio  plans.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 1]
```diff
@@ -103,11 +103,65 @@ The following table lists the available products and their **ProductId**.
...
+## Highlights of Visio Plan 1
+• Work virtually anywhere, anytime with the web version of Visio and 2 GB of OneDrive for Business cloud storage.
+
+• Start diagramming fast with an intuitive experience and simple UI.
+
+• Choose from dozens of professional templates, starter diagrams, and shapes.
+
+• Access and edit Visio files directly inside Microsoft Teams.*
+
+• Embed Visio diagrams into Power BI reports for unified perspectives.*
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
• Access and edit Visio files directly inside Microsoft Teams.

• Embed Visio diagrams into Power BI reports for unified perspectives.
```

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 2]
```diff
@@ -103,11 +103,65 @@ The following table lists the available products and their **ProductId**.
...
+
+• Enjoy everything in Plan 1 plus exclusive Plan 2 features such as export to Microsoft Flow or Word, Excel data visualizer, and PowerPoint slide snippets.
+
+• Connect diagrams to external data sources and embed Visio diagrams in Power BI.
+
+• Store and share your work on the web and access it from almost anywhere.
+
+• Enjoy around-the-clock support from Microsoft.
+
+## Purchase Project and Visio Plans through Microsoft 365 link
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
## Microsoft 365 links to purchase Project and Visio plans
```

File: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md [hunk 3]
```diff
@@ -103,11 +103,65 @@ The following table lists the available products and their **ProductId**.
...
+
+> **Project Plan 1** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-1?activetab=pivot:overviewtab)
+> 
+> **Project Plan 3** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-3?activetab=pivot:overviewtab)
+> 
+> **Visio Plan 1** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-1?activetab=pivot:overviewtab)
+> 
+> **Visio Plan 2** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-2?activetab=pivot:overviewtab)
+>
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md]:
```suggestion
**Project Plan 1** can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-1?activetab=pivot:overviewtab).

**Project Plan 3** can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/project/project-plan-3?activetab=pivot:overviewtab).

**Visio Plan 1** can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-1?activetab=pivot:overviewtab).

**Visio Plan 2** Can be purchased by using this [Microsoft 365 link](https://www.microsoft.com/microsoft-365/visio/visio-plan-2?activetab=pivot:overviewtab).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Review by RAJU2529 — COMMENTED** [review]
Please merge it

**Pull request merged**